lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -1013,11 +1018,10 @@ void ConfigBuilder::buildVsRegConfig(ShaderStage shaderStage, T *pConfig) {
     SET_REG_FIELD(&pConfig->vsRegs, PA_CL_VS_OUT_CNTL, VS_OUT_MISC_SIDE_BUS_ENA, true);
 
     if (gfxIp.major == 9) {
-    } else if (gfxIp.major == 10) {
-    } else
-      llvm_unreachable("Not implemented!");
+    } else {
+    }
 
-    if (gfxIp.major == 10 && gfxIp.minor >= 3) {
+    if (gfxIp >= GfxIpVersion({10, 3})) {
       SET_REG_GFX10_2_PLUS_FIELD(&pConfig->vsRegs, PA_CL_VS_OUT_CNTL, USE_VTX_VRS_RATE, useShadingRate);
```

Comment:
Probably doesn’t matter but it looks like this should check `gfxIp >= GfxIpVersion({10, 2})` for the GFX10_2_PLUS_FIELD.

Reply:
Yes, you are right. The situation here is we don't expose such GFX IP publicly as formal products. So we avoid using 10.2 check.